gen/sam/airdefensegroupgenerator.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, game: Game, ground_object: SamGroundObject) -> None:
 
         self.vg.name = self.group_name_for_role(self.vg.id, self.primary_group_role())
         self.auxiliary_groups: List[VehicleGroup] = []
+        self.heading = self.heading_to_conflict()
```

Comment:
`self.heading_to_conflict().degrees` I think, if this is setting a heading for a pydcs type.

Reply:
i takes a Heading not an Int/Float so it can stay like this :)